docparse/jsonschema.go

```diff
@@ -208,10 +208,23 @@ func fieldToSchema(prog *Program, fName string, ref Reference, f *ast.Field) (*S
 start:
 	switch typ := sw.(type) {
 
-	// Don't support interface{} for now. We'd have to add a lot of complexity
-	// for it, and not sure if we're ever going to need it.
+	// Interface, only useful for its description.
 	case *ast.InterfaceType:
-		return nil, errors.New("fieldToSchema: interface{} is not supported")
+		if len(f.Names) == 0 {
+			return nil, errors.New("fieldToSchema: interface{} is not supported")
```

Comment:
Is this error correct? I think that `len(f.Names) == 0` doesn't mean that the type is `interface{}`?

Reply:
No, it's wrong I was thinking of the actual type and that being empty with interface{}, I fix it and update test

Reply:
I've removed the check, doesn't seem to be a case where names would be nil. Added the empty inteface into the test output too

Reply:
> Doesn't seem to be a case where names would be nil

It's `0` when you're using it as an embedded struct (AFAIK it's never `nil`).